temporal-testing/build.gradle

```diff
@@ -17,9 +17,14 @@ dependencies {
     api project(':temporal-sdk')
     api project(':temporal-test-server')
 
+    // This dependency is included in temporal-sdk module as optional with compileOnly scope.
+    // To make things easier for users, it's helpful for the testing module to bring this dependency
+    // transitively as most users work with history jsons in tests.
```

Comment:
> most users work with history jsons in tests

_most_ is pretty strong here, but in general I support more required dependencies in testing libraries than runtime ones.